src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -376,7 +376,7 @@ def create_managed_disk(cmd, resource_group_name, disk_name, location=None,  # p
                 sku=_get_sku_object(cmd, sku), disk_size_gb=size_gb, os_type=os_type, encryption=encryption)
 
     if hyper_v_generation:
-        disk.hyper_vgeneration = hyper_v_generation
+        disk.hyper_v_generation = hyper_v_generation
```

Comment:
It seems that there are still many unnoticed but ubiquitous attribute/model/function-name error caused by track2 upgrading.
Is it necessary to check them thoroughly in advance rather than fix them after issues raising?

Reply:
You are right. It is a disadvantage of dynamic language. We can't detect the error until runtime.